File: Stegeriluminacion/frontend/src/app/_dto/admin/metrics.Dto.ts

export class MetricsDTO {

  processList: Array<String[]>;
  maxUsedConnections: Array<String[]>;
  waitTimeout: Array<String[]>;
  threadsConnected: Array<String[]>;
  threadsRunning: Array<String[]>;
  slowQueries: Array<String[]>;
  top10StatementsByLongestAverageRunTime: Array<String[]>;
  performanceSchemas: Array<String[]>;
  statementsWithRuntimesIn95thPercentile: Array<String[]>;
  statementsWithErrorsOrWarnings: Array<String[]>;
  maxConnections: Array<String[]>;

}

File: dto/src/main/java/dto/admin/MetricsDTO.java
package dto.admin;

import java.util.List;

import lombok.Data;

@Data
public class MetricsDTO {
	
	List<Object[]> processList;
	List<Object[]> maxUsedConnections;
	List<Object[]> waitTimeout;
	List<Object[]> threadsConnected;
	List<Object[]> threadsRunning;
	List<Object[]> slowQueries;
	List<Object[]> top10StatementsByLongestAverageRunTime;
	List<Object[]> performanceSchemas;
	List<Object[]> statementsWithRuntimesIn95thPercentile;
	List<Object[]> statementsWithErrorsOrWarnings;
	List<Object[]> maxConnections; 
}
